### ISSUE TITLE
this will be great if it support vue files

### ISSUE THREAD

**Issue by LouisaNikita** [AtomLinter/linter-stylelint#330]
i cant use stylelint in .vue files~its will be more mighty if support .vue

**Comment by pingshunhuangalex** [comment]
I'm been wanting this for so long... Just wonder if there is any update on this. If not, if there is any workaround I could use?

**Comment by MaxMilton** [comment]
I'd like to see a configurable list of scopes to run the linter on. *linter-eslint* is a good example of this done well.

**Comment by ismay** [comment]
Same goes for linting js and jsx with https://github.com/styled-components/stylelint-processor-styled-components. Seems to not be possible with linter-stylelint currently, but it works with stylelint's interface and ale on neovim.

Would be great to be able to do this with linter-stylelint.

**Comment by Arcanemagus** [comment]
Closing as a duplicate of https://github.com/AtomLinter/linter-stylelint/issues/235 as that would need to be done before this could be implemented, and doing that properly would mean that you could set this up yourself.
